src/mysql_protocol.erl

```diff
@@ -423,6 +440,8 @@ fetch_response(SockModule, Socket, Timeout, Proto, Acc) ->
                     parse_ok_packet(Packet);
                 ?error_pattern ->
                     parse_error_packet(Packet);
+                <<251:8, FileName/binary>> -> 
```

Comment:
Please add a macro for 251 where the other packet tags are defined, with an appropriate name and comment if needed.